setup/common/040_ValidateSignCert.rb

```diff
@@ -21,7 +21,7 @@
   hosts.each do |host|
     ssldir = on(host, puppet('agent --configprint ssldir')).stdout.chomp
     # preserve permissions for master's ssldir so puppetserver can read it
-    on(host, "rm -rf '#{ssldir}/*'")
+    on(host, "rm -rf '#{ssldir}/'*")
```

Comment:
did some tests on Windows/Linux with directory containing spaces, we need to keep `#{ssldir}` in single thicks